src/main/sensors/acceleration.c

```diff
@@ -215,7 +216,22 @@ void updateAccelerationReadings(rollAndPitchTrims_t *rollAndPitchTrims)
     }
 
     for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-        accSmooth[axis] = lrintf(accFilterApplyFn(&accFilter[axis], (float)accADCRaw[axis]));
+      accSmooth[axis] = accADCRaw[axis];
+    }
+    
+    if (accelerometerConfig()->acc_cut_hz) {
+        if (!accFilterInitialised) {
+            if (accSamplingInterval) {  /* Initialisation needs to happen once sample rate is known */
+                accelerationFilterInit(accelerometerConfig()->acc_cut_hz);
+                accFilterInitialised = true;
+            }
+        }
+
+        if (accFilterInitialised) {
+            for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
+                accSmooth[axis] = lrintf(biquadFilterApply(&accFilter[axis], (float)accSmooth[axis]));
+            }
+        }
```

Comment:
These changes are no longer required now that `accSamplingInterval` is set properly.